mscore/scoreview.cpp

```diff
@@ -3522,6 +3530,15 @@ void ScoreView::noteEntryButton(QMouseEvent* ev)
       ChordRest* cr = _score->inputState().cr();
       if (cr)
             adjustCanvasPosition(cr, false);
+
+      // get the position of the cursor rect for the next noteentry
```

Comment:
This is probably the code that make the autoadvance work. This is unrelated to the shadown notes, not sure we want to move the mouse cursor if the user doesn't move her mouse...